modules/ibi-install-lcao-cli.adoc

```diff
@@ -0,0 +1,111 @@
+// Module included in the following assemblies:
+// * edge_computing/ibi-image-based-install.adoc 
+
+:_mod-docs-content-type: PROCEDURE
+[id="ibi-install-lcao-cli_{context}"]
+= Installing the {lcao} by using the CLI
+
+You can use the OpenShift CLI (`oc`) to install the {lcao} from the 4.15 Operator catalog.
```

Comment:
We shouldn't mention 4.15

Reply:
At the time of writing, I believe the Lifecycle Agent was only present in the 4.15 Operator catalog. The [software prerequisites](https://75088--ocpdocs-pr.netlify.app/openshift-enterprise/latest/edge_computing/image-base-install/ibi-preparing-for-image-based-install#ztp-image-based-upgrade-prereqs_ibi-understanding-image-based-install) say 4.16 for LCA. @eranco74 - can you clarify, do we need to mention a version for LCA?

Reply:
This should match the IBU docs (I'm pretty sure it should be 4.16).
In general the seed generation is the same for IBU and IBI

Reply:
The minimum software requirements say 4.16 for LCA in the IBU [docs](https://74025--ocpdocs-pr.netlify.app/openshift-enterprise/latest/edge_computing/image_based_upgrade/cnf-understanding-image-based-upgrade#ztp-image-based-upgrade-cluster-validated-software_understanding-image-based-upgrade). 

Perhaps my lack of understanding, but why doesn't LCA show up in the Operator catalog when I try to install LCA in a 4.16 clusterbot cluster. But LCA is availalbe in a 4.15 clusterbot cluster.